docs/developers/contributing.rst

```diff
@@ -10,25 +12,27 @@ use your contributions. In effect, you’re giving us a license, but you still
 own the copyright — so you retain the right to modify your code and use it in
 other projects.
 
-The agreement can be found, and signed, here:
-https://ubuntu.com/legal/contributors
+Please review and sign the `Canonical contributor licence agreement`_.
 
-## Contributor guidelines
 
-- Each commit should be a logical unit.
-- Each commit should pass tests individually to allow bisecting.
-- Each commit must be signed.
-- The commit message should focus on WHY the change is necessary, we get the
+Contributor guidelines
+----------------------
+
+* Each commit should be a logical unit.
+* Each commit should pass tests individually to allow bisecting.
+* Each commit must be signed.
+* The commit message should focus on WHY the change is necessary, we get the
   what and how by looking at the code.
-- Include a Signed-off-by header in the commit message.
-- MicroOVN makes use of the GitHub Pull Request workflow.  There is no
+* Include a Signed-off-by header in the commit message.
+* MicroOVN makes use of the GitHub Pull Request workflow.  There is no
   meaningful way to manage interdependencies between GitHub PRs, so we expect
   dependent changes proposed in a single PR reviewed and merged as separate
   commits.
-- A proposal for change is not complete unless it contains updates to
+* A proposal for change is not complete unless it contains updates to
   documentation and tests.
 
-## Tests
+Tests
```

Comment:
Now that this page is part of the documentation, it would probably be better to link this section to `docs/developers/testing.rst`, instead of repeating the information.

Reply:
That is a good idea, slight complication for users reading this from the repository, but I think the comment pointing at the location of the docs which is only visible for someone reading the source would solve that nicely.